CMakeLists.txt

```diff
@@ -573,15 +572,17 @@ if(CIRCT_SLANG_FRONTEND_ENABLED)
       set(CMAKE_CXX_FLAGS "")
     endif ()
     set(BUILD_SHARED_LIBS OFF)
+    set(SLANG_USE_MIMALLOC OFF)
     FetchContent_MakeAvailable(slang)
 
+    set(CMAKE_CXX_STANDARD 20)
+    set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
Not sure where it is most appropriate to set this, input would be much appreciated.